src/core/common.hh

```diff
@@ -120,6 +120,10 @@
 #define SSC_VERSION_HASH ""
 #endif
 
+#ifndef PORT
+#define PORT 0
```

Comment:
You might want a better default here?

Reply:
I remember reading that iOS has certain ports that are known not to run anything, iirc, 9777 is one, but I cant be sure, let me see if I cant find that information somewhere.

Reply:
let's go with it for now, but I think that `0` could be a problem for that reason in particular, there are a lot of well known programs that can come online and expect a certain port